.readthedocs.yml

```diff
@@ -1,11 +1,16 @@
-requirements_file: doc/requirements.txt
+version: 2
 
-build:
-    image: latest
+sphinx:
+  configuration: doc/conf.py
 
 python:
-    version: 3.8
+  install:
+    - requirements: doc/requirements.txt
+    - method: pip
+      path: .
+  system_packages: true
 
-# Don't build any extra formats
-formats:
-    - none
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
```

Comment:
Should we also bump the python version? I think we'll be dropping python 3.8 support in the next release.

Reply:
Bumped!